ebml_matroska.xml

```diff
@@ -215,10 +215,10 @@
     <documentation lang="en">Set if the track is usable. (1 bit)</documentation>
   </element>
   <element name="FlagDefault" path="1*1(\Segment\Tracks\TrackEntry\FlagDefault)" cppname="TrackFlagDefault" id="0x88" type="uinteger" minOccurs="1" maxOccurs="1" minver="1" default="1" range="0-1">
-    <documentation lang="en">Set if that track (audio, video or subs) SHOULD be active if no language found matches the user preference. (1 bit)</documentation>
+    <documentation lang="en">For audio or video, set if that track SHOULD be active if no language found matches the user's preference; this can be used to specify the original content language. For subtitles, set if that track SHOULD be chosen if there are multiple tracks which match the user's language preferences. (1 bit)</documentation>
```

Comment:
We can't say it could mean something else too (original content), That would be another flag and you can't have data that are left to interpretation depending on what you think the file means.
I don't see why subtitles should be treated differently than audio or video.

Reply:
That sounds better, yes.